akka-persistence/src/main/scala/akka/persistence/SnapshotProtocol.scala

```diff
@@ -11,14 +11,14 @@ import scala.runtime.AbstractFunction3
  * @param persistenceId id of persistent actor from which the snapshot was taken.
  * @param sequenceNr sequence number at which the snapshot was taken.
  * @param timestamp time at which the snapshot was saved, defaults to 0 when unknown.
- * @param meta a journal can optionally support persisting metadata separate to the domain state
+ * @param metadata a journal can optionally support persisting metadata separate to the domain state, used for active active support
  */
 @SerialVersionUID(1L)
 final class SnapshotMetadata(
     val persistenceId: String,
     val sequenceNr: Long,
     val timestamp: Long,
-    val meta: Option[Any])
+    val metadata: Option[Any])
```

Comment:
I had this on a tab and forgot to send the comment. 

Now I understand why it had a different name. We have already a metadata and now we are adding an optional field to it. Strange to have `metadata.metadata`. 

I don't now what would make it more clear. 

* `extras` - `metadata.extras`
* `optional` - `metadata.optional`
* `modelMetadata` - `metadata.modelMetadata`
* `replica` - `metadata.replica`

We can also leave like that though.